src/lib/nodemailer.js

```diff
@@ -0,0 +1,71 @@
+const promisify = require("util").promisify;
+const nodemailer = require("nodemailer");
+const EMAIL_SIGNATURE = require("../services/emails/signature");
+
```

Comment:
The name of the file should maybe be switched to `email.js` or `email_service.js` or something similar.

Having the name of the library as the file name might introduce some confusion as to whether the service or nodemailer itself is being used directly when browsing the code.